utilities/decrypt_data/export_postgres_to_excel_chunks.py

```
@@ -0,0 +1,585 @@
+def serialize_row(row):
+    return {
+        k: (json.dumps(v, ensure_ascii=False) if isinstance(v, (dict, list)) else v)
+        for k, v in row.items()
+    }
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(__name__)
+
+
+class PostgreSQLExporter:
+    def __init__(self, dsn: str):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)